### DISCUSSION TITLE
Real-time detection results streaming with a full screen

### DISCUSSION THREAD

**Discussion by alesvaz** [ultralytics/yolov5#10315] [Q&A] [answered]
Hi,
I am quite new to yolov5 and neural networks in general. At work, I have been asked to show in real-time the results of the detection (using a raspberry as webcam, with the pre-trained model loaded in the pc) on an external screen: how can I change the resolution of the detection results? Now they occupy just one quarter of the screen, I would like the output to be full screen (I changed the img-size parameter but nothing seemend to happen).

Any tip would be greatly appreciated. Thank you.

**Comment by glenn-jocher** [comment, marked as answer]
@alesvaz --imgsz parameter governs inference shape, not image shape. The shape of your stream is really outside the context of YOLOv5.
